### ISSUE TITLE
How to add depth supervised loss?

### ISSUE THREAD

**Issue by rockywind** [Brummi/BehindTheScenes#11]
Hi, 
I added the depth supervised loss in [the config file](https://github.com/Brummi/BehindTheScenes/blob/b866ad031e0a7c81c3a6067c99b6c604aae2c6fa/configs/exp_kitti_360.yaml#L69).
But, I found the depth loss starts small and oscillates up.
The visualization of depth map was also poor.
![image](https://user-images.githubusercontent.com/14838781/229669336-e3814a7a-4749-43ae-ab8b-6190edc10bea.png)
![image](https://user-images.githubusercontent.com/14838781/229670211-9d9469ed-cbe7-4c3b-8c1e-89b3bcd9d3b9.png)

![image](https://user-images.githubusercontent.com/14838781/229670262-c6447082-bd9e-47ca-a26c-5f412d14fff5.png)

**Comment by Brummi** [comment]
The `lambda_depth_reg` hyperparameter scales the depth regularization (which basically enforces the depth map to be smooth).
We experimented, whether this might improve training stability, but found that it is not necessary. 
As you can see, the depth maps become more blurry / smooth.

All the loss components not mentioned in the paper were deemed to be not beneficial for the performance for the model. I left them in the code as it might be interesting for others to experiment with these.
